src/main.js

```diff
@@ -122,14 +112,9 @@ function viewSaved() {
   randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
-
-  renderSavedCovers();
-};
-
-function renderSavedCovers() {
   for (var i = 0; i < savedCovers.length; i++) {
     savedCoversSection.innerHTML += `
-      <section class="mini-cover" id="${savedCovers[i].id}">
+      <section class="main-cover" id="${savedCovers[i].id}">
```

Comment:
Should this be `main` or `mini`? Believe saved covers are supposed to render at a smaller size than the main?

Reply:
Will change to mini.